### DISCUSSION TITLE
How to NgRest CRUD by fields type or auto generate relationships

### DISCUSSION THREAD

**Discussion by fanlt** [luyadev/luya#2142] [Questions & Problems]
For example enum fields type ,can auto generate to select or checkbox?

**Comment by nadar** [comment]
https://luya.io/guide/ngrest-plugins

**Comment by fanlt** [comment]
This must add ngRestExtraAttributeTypes in model classs,How to auto generate  it!

**Reply by nadar** [reply]
Its not possible to auto generate. Using the `admin/crud` command will create the models, but relations must be added by yourself.
